### PR TITLE
feat: ts refactor, filter stack trace, update pretty

### PR DESCRIPTION
**Pull request by z4o4z**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
   "author": "Frank Gu <frank@voiceflow.com>",
   "dependencies": {
     "@types/express-pino-logger": "^4.0.2",
-    "@types/pino": "^6.3.0",
+    "@types/pino": "6.3.4",
     "@voiceflow/pino-pretty": "^3.5.0",
```
Comment (frankgu968) [inline package.json:9]: Update this one after merging the pino-pretty changes

**Comment by frankgu968** [comment]
Can you please fix the call stack test(s) as well? I think the paths changed so you'll have to update the references

**Review by DecathectZero — DISMISSED** [review]

**Review by frankgu968 — CHANGES_REQUESTED** [review]
@z4o4z I see that you have removed the Caller identification code, but I don't see any refactor into TS. How is the callstack identified now?? Is it part of Pino's log??

**Review by frankgu968 — APPROVED** [review]

**Pull request merged**